### PR TITLE
Refs #10962 - limit foreman tasks in rpm specs to < 0.7.0

### PR DESCRIPTION
**Pull request by iNecas**
While preparing the new release of foreman tasks with Dynflow, I've hit an
issue when testing new release with old Katello, as the requirements in the
gemspec don't match the spec.

Another PR will follow updating the Katello to work with new foreman-tasks/dynflow
version, that is being prepared.

### CODE REVIEW

**Comment by ehelms** [comment]
Can we limit it to '< 1.0.0' ? And what about the gemspec, does it need updating?

**Comment by iNecas** [comment]
The gemspec already has the limit included https://github.com/Katello/katello/blob/master/katello.gemspec#L31 , we are still in the 0.y.z phase according  to http://semver.org/, where the minor number also can include some backward incompatibilities, which will be case also this time. The 0.7.0 will be mostly compatible for all plugins, but Katello will need an update due to ListenOnCandlepinEvent: it would work even without any changes, but would produce some unnecessary error messages.

Long story short, we are not releasing foreman-tasks 1.0.0, but Katello will need some minor updates to get the most out of the new release

**Comment by iNecas** [comment]
@ehelms  Good to merge? This is blocking the relese process for new foremant-tasks/dynflow

**Comment by ehelms** [comment]
ACK
